docs/UserGuide.md

```diff
@@ -141,80 +141,122 @@ Format: `clear`
 
 #### Here's how it would look like in the app:
+TO UPDATE IMAGE AFTER FINAL UPDATE TO APPLICATION!!!
 ![add Command](images/addSupplierCommand.png)
 
+#### Valid email address format
+
+Valid email addresses are of the format: `local-part@domain`.  
+Both `local-part` and `domain` must start and end with alphanumeric characters.  
+Special characters cannot be used consecutively.
+
+`local-part` must be alphanumeric, or these special characters: `+.-_`.
+
+`domain` must be alphanumeric, or these special characters: `.-`.  
+`domain` can be separated into multiple parts with `.`, and the ending part must be at least 2 alphanumeric characters.  
+- Each part must start and end with alphanumeric characters.
+- e.g. `example.com` is separated into two parts, `example` and `com`, and is a valid `domain`.
+
+Valid email address examples:
+- john.doe@example.com
+- user123@my-website.com
+- first.last@school.edu.sg
+
+Invalid email address examples:
+- john.@example.com (`local-part` (i.e. `john.`) cannot end in a `.`)
+- john--doe@example.com (Special character `-` cannot be used consecutively)
+- john.doe@example.c (Final `domain` part (i.e. `c`) must have at least 2 alphanumeric characters)
+
+#### Valid company name format
+
+Valid company names must be alphanumeric or punctuation characters, and spaces are allowed.  
+Punctuation characters include ``!"#$%&'()*+,-./:;<=>?@[\]^_\\`{|}~``
+
 ### Listing all suppliers: `list -s`
 
-Shows a list of all suppliers in VendorVault. (The delivery list will not be affected)
+Shows a list of all suppliers in VendorVault. The delivery list will not be affected.
 
 Format: `list -s`
 
 <box type="warning" seamless>
 
 **Warnings**:
-- At least one space between list and -s
+- At least one space is needed between `list` and `-s`.
 - No other parameters should be given for this command.
-  </box>
 
-### Deleting a supplier : `delete -s`
+</box>
 
-The `delete -s` command is used to delete a supplier from the list of suppliers in VendorVault.
+### Deleting a supplier : `delete -s`
 
+Deletes a supplier from the list of suppliers in VendorVault.
 
 Format: `delete -s INDEX`
-- `INDEX`: The index of the supplier in the list.
+
+<box type="details" seamless>
+
+Parameters:
+
+- `INDEX`: The index of the supplier to be deleted in the displayed list. Must be a positive numeric number.
+
+</box>
 
 <box type="warning" seamless>
 
 **Warnings**:
-- A spacing between `delete` and `-s` is compulsory
-- No duplicate prefix can be used
-</box>
+- At least one space is needed between `list` and `-s`.
+- Only one supplier can be deleted by one command.
 
-#### Example
-To delete the supplier at index 3:
+</box>
 
-    delete -s 3
+Example:
+`delete -s 3`
 
-A success message will be displayed if the supplier is successfully deleted.
+Expected output:
+Supplier at index 3 is deleted, assuming it existed initially. Otherwise, an error message will be shown.
```

Comment:
Are we allowed to change this status message?

Reply:
where?